### DISCUSSION TITLE
Adding styled-components to Remix 1.7.1

### DISCUSSION THREAD

**Discussion by ghost** [remix-run/remix#4240] [Q&A]
I'm new to Remix and I'm curious if someone has an example on how to add styled components to Remix. The guide on https://remix.run/docs/en/v1/guides/styling#css-in-js-libraries seems to cover an older version of Remix (I'm currently on `1.7.1`)? I'm referring to this part of `entry.server.txs` :

`return new Response("<!DOCTYPE html>" + markup, {
    status: responseStatusCode,
    headers: responseHeaders,
  });`

Where `"<!DOCTYPE html>" + markup` now is replaced with `body` from `const body = new PassThrough()`. Since I'm not really understanding what is going on here, I'm not sure how to solve this on my own. 

Could anyone help/explain? Thanks!

**Comment by KasparRosin** [comment]
styled-components does not yet support stream ssr.
https://github.com/styled-components/styled-components/issues/3658
